website/src/pages/Home/workloads.tsx

```diff
@@ -0,0 +1,290 @@
+import React, { useState, useEffect } from "react";
+import { Typography, Link, Box, Button, Paper, withStyles, Tooltip } from '@material-ui/core';
+import { useTranslation } from "react-i18next";
+import useStyles from './styles';
+import { EXTERNAL_LINKS, VIEW_PORT } from "../../constants";
+import Asciinema from '../../components/Asciinema';
+import { useViewport } from "../../hooks/viewportWidth";
+import { useExternalStyles } from "../../hooks/useExternalStyles";
+import { useScript } from '../../hooks/useScript';
+
+export const Workloads: React.FC = () => {
+
+    useExternalStyles('css/asciinema-player.css');
+    useExternalStyles('css/custom-asciinema-player.css');
+    const asciinemaScript = useScript('js/asciinema-player.js');
+
+     const { t } = useTranslation();
+     const classes = useStyles();
+
+     const [copyCommand, setCopyCommand] = useState({
+        text: 'helm install stable/openebs --name openebs --namespace openebs',
+        status: 'Copy to clipboard'
+    });
+
+    const [isTabletView, setIsTabletView] = useState<boolean>(false);
+    const { width } = useViewport();
+
+    useEffect( () => {
+        window.innerWidth <= VIEW_PORT.LAPTOP_BREAKPOINT ? setIsTabletView(true) : setIsTabletView(false);
+    },[width])
+
+     const LightTooltip = withStyles((theme) => ({
```

Comment:
```suggestion
     const lightTooltip = withStyles((theme) => ({
```

Reply:
`LightTooltip` is a reactElement, better to keep `pascal case`